encrypted-media-respec.html

```diff
@@ -600,6 +600,13 @@
             multiple independent profiles for a single account.
           </p>
         </dd>
+        <dt id="valid-media-mime-type">Valid Media MIME Type</dt>
+        <dd>
+          <p>
+            A media <a def-id="mime-types">MIME type</a> [[!HTML51]] represents the type of the <a def-id="media-resource">media resource</a>. Its MUST be a <a def-id="valid-mime-type">valid media MIME type</a> [[!HTML51]]. 
```

Comment:
Where does `valid-mime-type` link to and why is the link text "Valid media MIME type" which is the title of this section. Is `valid-mime-type` supposed to be a link out to HMTL5 ?